### PR TITLE
Add support for bazel and kokoro.

### PR DESCRIPTION
**Pull request by jverkoey**
**DO NOT MERGE** until kokoro signs this PR.

### CODE REVIEW

**Comment by jverkoey** [comment]
This is ready for review.

File: Sources/MDFInternationalization.h [hunk 1]
```diff
@@ -16,9 +16,9 @@
 
 #import <UIKit/UIKit.h>
 
-#import <MDFInternationalization/MDFRTL.h>
-#import <MDFInternationalization/UIImage+MaterialRTL.h>
-#import <MDFInternationalization/UIView+MaterialRTL.h>
+#import "MDFRTL.h"
+#import "UIImage+MaterialRTL.h"
+#import "UIView+MaterialRTL.h"
```
Comment (ianegordon) [inline Sources/MDFInternationalization.h]: We are following Apple's guidelines for frameworks here.
Is this change required for kokoro?

**Review by ianegordon — CHANGES_REQUESTED** [review]

File: Sources/MDFInternationalization.h (hunk 1 above)
Comment (jverkoey) [reply Sources/MDFInternationalization.h]: Yes it's unfortunately the lowest common denominator for building succesfully across CocoaPods/Xcode/Bazel.

**Comment by jverkoey** [comment]
The script now fixes header imports.

**Comment by jverkoey** [comment]
TODO:

- [ ] WORKSPACE should use `http_archive` instead of `git_repository`
- [ ] Bump build_bazel_rules_apple to 0.1.0.

**Review by ianegordon — APPROVED** [review]
